src/browser/index.js

```diff
@@ -6,12 +6,15 @@ define((require, exports, module) => {
 
   'use strict';
 
+  require('./github'); // Pull updates from github
```

Comment:
I would much rather prefer if this exported something along this lines:

```js
const update = () => {
  window.dispatchEvent(new CustomEvent("app-update-available"));
  require('./github').updateAvailable().then(update);
}

require('./github').updateAvailable().then(update);
```

So there would be a way to start / abort polling if necessary.